### PR TITLE
Add ARM64_GRP_PAC group for Pointer Authentication

### PR DESCRIPTION
**Pull request by elp0t0r1c0**
Hi,

I've added the group `ARM64_GRP_PAC` for grouping instructions relative to [PAC](https://www.qualcomm.com/media/documents/files/whitepaper-pointer-authentication-on-armv8-3.pdf).
Here is the list of the instructions I added to this new group:
 * [ARM64_INS_AUTDA (AUTDA), ARM64_INS_AUTDZA (AUTDZA)](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0801g/ycn1476202675817.html) 
 * [ARM64_INS_AUTDB (AUTDB), ARM64_INS_AUTDZB (AUTDZB)](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0801g/epx1476202676637.html)
 * [ARM64_INS_AUTIA (AUTIA), ARM64_INS_AUTIA1716 (AUTIA1716), ARM64_INS_AUTIASP (AUTIASP), ARM64_INS_AUTIZA (AUTIZA), ARM64_INS_AUTIAZ (AUTIAZ)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/rmi1476202677017.html)
 * [ARM64_INS_AUTIB (AUTIB), ARM64_INS_AUTIB1716 (AUTIB1716), ARM64_INS_AUTIBSP (AUTIBSP), ARM64_INS_AUTIZB (AUTIZB), ARM64_INS_AUTIBZ (AUTIBZ)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/lru1476202677407.html)
 * [ARM64_INS_BLRAA (BLRAA), ARM64_INS_BLRAAZ (BLRAAZ), ARM64_INS_BLRAB (BLRAB), ARM64_INS_BLRABZ (BLRABZ)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/zwh1476202686019.html)
 * [ARM64_INS_BRAA (BRAA), ARM64_INS_BRAAZ (BRAAZ), ARM64_INS_BRAB (BRAB), ARM64_INS_BRABZ (BRABZ)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/hgj1476202687129.html)
 * [ARM64_INS_ERETAA (ERETAA), ARM64_INS_ERETAB (ERETAB)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/nps1476202716430.html)
 * [ARM64_INS_PACDA (PACDA), ARM64_INS_PACDZA (PACDZA)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/mlg1476202744441.html)
 * [ARM64_INS_PACDB (PACDB), ARM64_INS_PACDZB (PACDZB)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/ggo1476202744811.html)
 * [ARM64_INS_PACGA (PACGA)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/fbm1476202745161.html)
 * [ARM64_INS_PACIA (PACIA), ARM64_INS_PACIZA (PACIZA), ARM64_INS_PACIA1716 (PACIA1716), ARM64_INS_PACIASP (PACIASP), ARM64_INS_PACIAZ (PACIAZ)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/vco1476202745841.html)
 * [ARM64_INS_PACIB (PACIB), ARM64_INS_PACIZB (PACIZB), ARM64_INS_PACIB1716 (PACIB1716), ARM64_INS_PACIBSP (PACIBSP), ARM64_INS_PACIBZ (PACIBZ)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/mpg1476202746202.html)
 * [ARM64_INS_RETAA (RETAA), ARM64_INS_RETAB (RETAB)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/osg1476202748462.html)
 * [ARM64_INS_XPACD (XPACD), ARM64_INS_XPACI (XPACI), ARM64_INS_XPACLRI (XPACLRI)](http://infocenter.arm.com/help/topic/com.arm.doc.dui0801g/she1476202785923.html)

(total: 44)

If I have forgotten some instructions, tell me, I'll add them to the group.

### CODE REVIEW

File: arch/AArch64/AArch64Mapping.c [hunk 1]
```diff
@@ -430,6 +430,7 @@ static const name_map group_name_maps[] = {
 	{ ARM64_GRP_PRIVILEGE, "privilege" },
 	{ ARM64_GRP_INT, "int" },
 	{ ARM64_GRP_BRANCH_RELATIVE, "branch_relative" },
+	{ ARM64_GRP_PAC, "Pointer Authentication" },
```
Comment (aquynh) [inline arch/AArch64/AArch64Mapping.c]: just to be consistent, can you keep the name in lower case?

**Comment by elp0t0r1c0** [comment]
Done

**Comment by patateqbool** [comment]
Mé nn?

**Comment by aquynh** [comment]
merged, thanks!

**Pull request merged**